Configuration/config.py

```
@@ -0,0 +1,16 @@
+# firebase admin 
+import firebase_admin
+# credentials,auth,firestore 
+from firebase_admin import credentials,auth,firestore 
+
+    
+class Configuration:
+    def __init__(self):
+        cred=credentials.Certificate(r"E:\Budget-Manager\API-Gateway\Configuration\database.json")
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)